.github/workflows/daily-tests.yml

```diff
@@ -10,14 +10,14 @@ jobs:
   ci1:
     uses: ./.github/workflows/daily-test-build.yml
     with:
-      libtiledb_version: '2.26.0'
+      libtiledb_version: '2.27.0'
```

Comment:
Should we parameterize this at the top of the file?

Reply:
noted